### PR TITLE
Add language settings (+ settings menu refactoring)

### PR DESCRIPTION
**Pull request by aurelenc**

### CODE REVIEW

File: locales/locale_en.po [hunk 1]
```diff
@@ -25,6 +22,30 @@ msgstr ""
...
+msgstr ""
+
+msgid "Sfx"
+msgstr ""
+
+msgid "Language"
+msgstr ""
+
+msgid "English"
+msgstr ""
```
Comment (MisterPeModder) [inline locales/locale_en.po:38]: The language names should appear in their original names.

For example, imagine if we were to add Chinese support, a non-Chinese speaking user would not be able to know which language is which if they switch to Chinese.

File: locales/locale_fr.po [hunk 2]
```diff
@@ -25,6 +22,30 @@ msgstr "Volume"
...
+msgstr "Muet/Non-muet"
+
+msgid "Sfx"
+msgstr "Effets sonores"
+
+msgid "Language"
+msgstr "Langue"
+
+msgid "English"
+msgstr "Anglais"
```
Comment (MisterPeModder) [inline locales/locale_fr.po:38]: The language names should appear in their original names.

For example, imagine if we were to add Chinese support, a non-Chinese speaking user would not be able to know which language is which if they switch to Chinese.

**Review by AndreasLrx — APPROVED** [review]
Changing the language set the selection to the 'Back' button. If it doesn't take too many time fix it. Otherwise it doesn't matter since we don't change language everytime.

**Review by MisterPeModder — CHANGES_REQUESTED** [review]

File: locales/locale_en.po (hunk 1 above)
Comment (aurelenc) [reply locales/locale_en.po:38]: that makes sense

Comment (aurelenc) [reply locales/locale_en.po:38]: fixing it now

**Review by MisterPeModder — APPROVED** [review]

**Pull request merged**